### PR TITLE
DO NOT MERGE: hacky breaking changes to make it talk to a real backend

### PR DESCRIPTION
**Pull request by chrisns**
This breaks all the FE tests, it was done purely as an experiement to see the frontend talking to backend

### CODE REVIEW

**Comment by sassquad** [comment]
I've performed the necessary style rule changes to enable the pages to render as intended. These should surface in the latest commit to this branch.

**Comment by sassquad** [comment]
Some further style amends to get the Stats page more inline with Karwai's final designs added to the latest commit on this branch.

**Comment by fulljames** [comment]
@chrisns this should be mergeable now.

File: assets/scss/styles.scss [hunk 1]
```diff
@@ -21,29 +23,53 @@ $padding: 15px;
...
+          width: 90%;
+          float: left;
+        }
+      }
+    }
+    .header-proposition {
+      padding-top: 0;
+      #proposition-name {
+        font-family: "nta", Arial, sans-serif;
+        font-size: 36px;
```
Comment (chrisns) [inline assets/scss/styles.scss]: Please don't use font sizes in px inherit from a header if thats what you're trying to do?

File: assets/scss/styles.scss [hunk 2]
```diff
@@ -21,29 +23,53 @@ $padding: 15px;
...
+        line-height: 1.25;
+        font-weight: 400;
```
Comment (chrisns) [inline assets/scss/styles.scss]: you're setting the `font-weight` twice:

File: assets/scss/styles.scss [hunk 3]
```diff
@@ -21,29 +23,53 @@ $padding: 15px;
...
+        text-transform: none;
+        font-weight: bold;
+        color: white;
```
Comment (chrisns) [inline assets/scss/styles.scss]: $white if you absolutely must?

File: assets/scss/styles.scss [hunk 4]
```diff
@@ -21,29 +23,53 @@ $padding: 15px;
...
+        .header-proposition {
+          width: 90%;
+          float: left;
+        }
+      }
+    }
+    .header-proposition {
+      padding-top: 0;
+      #proposition-name {
+        font-family: "nta", Arial, sans-serif;
```
Comment (chrisns) [inline assets/scss/styles.scss]: is this not available as something from the gov patterns?

File: assets/scss/styles.scss [hunk 5]
```diff
@@ -53,14 +79,14 @@ h1 {
 }
 
 /* Dashboard */
-#centres {
+.centre_data {
   padding: 0;
   h2 {
     padding-bottom: $padding+5;
     line-height: 24px;
```
Comment (chrisns) [inline assets/scss/styles.scss]: please don't use px

File: assets/scss/styles.scss [hunk 6]
```diff
@@ -53,14 +79,14 @@ h1 {
...
   }
   .panel {
-    background-color: #DEE0E2;
+    background-color: $background-color;
     margin: 0 50px 35px;
```
Comment (chrisns) [inline assets/scss/styles.scss:87]: please don't use px

File: assets/scss/styles.scss [hunk 7]
```diff
@@ -126,69 +152,52 @@ h1 {
 }
 
 /* Stats page */
-#stats {
+.stat_data {
+  h2 {
+    @include bold-24;
+  }
   table {
+    margin: 0 50px 50px;
```
Comment (chrisns) [inline assets/scss/styles.scss]: please don't use px

File: assets/scss/styles.scss [hunk 8]
```diff
@@ -126,69 +152,52 @@ h1 {
...
-    width: 100%;
-    margin-bottom: 50px;
+    box-sizing: border-box;
     thead {
       tr {
         th,
         td {
-          padding-top: 0;
-          padding-bottom: 16px;
+          padding: 0 1.25em 16px 0;
```
Comment (chrisns) [inline assets/scss/styles.scss]: mix of px + em?

File: assets/scss/styles.scss [hunk 9]
```diff
@@ -126,69 +152,52 @@ h1 {
...
-        }
-        th:last-child {
-          text-align: right;
+          &.numeric {
+            @include bold-19($tabular-numbers: true);
+            text-align: right;
+          }
+          &.centre_updated {
+            @include core-16;
+            color: red;
```
Comment (chrisns) [inline assets/scss/styles.scss]: `$red` if you must

Comment (chrisns) [inline assets/scss/styles.scss]: is there no `highlight` or equivalent you could use?

File: assets/scss/styles.scss [hunk 10]
```diff
@@ -126,69 +152,52 @@ h1 {
...
+            text-align: right;
+          }
+          &:last-child {
+            width: 10%;
```
Comment (chrisns) [inline assets/scss/styles.scss]: will cause overlap issues if the table is rendered too narrow

File: assets/scss/styles.scss [hunk 11]
```diff
@@ -126,69 +152,52 @@ h1 {
...
-              line-height: 1.25;
-            }
-          }
-        }
         th,
         td {
           @include core-19;
           padding: 0.75em 1.25em 0.5625em 0;
           text-align: left;
           color: #0b0c0c;
```
Comment (chrisns) [inline assets/scss/styles.scss]: is there nothing in the gov stylesheet for this colour? if not - should we even use it?

File: assets/scss/styles.scss [hunk 12]
```diff
@@ -126,69 +152,52 @@ h1 {
...
-          border-bottom: 1px solid #6f777b;
           width: 25%;
           &:first-child {
             width: 50%;
```
Comment (chrisns) [inline assets/scss/styles.scss:173]: will cause wrapping/overflow issues in narrow viewports

File: assets/templates/stat.html [hunk 13]
```diff
@@ -1,12 +1,13 @@
-<div class="column-two-thirds">
+<div class="column-three-quarters">
+  <h2 class="centre_name">{{name}}</h2>
   <table>
     <thead>
       <tr>
-        <th class="centre_name" scope="col">{{name}}</th>
+        <th scope="col">&nbsp;</th>
```
Comment (chrisns) [inline assets/templates/stat.html]: why do we need an empty column everywhere?

File: assets/scss/styles.scss (hunk 1 above)
Comment (sassquad) [inline assets/scss/styles.scss]: Tom and I agree that using fixed size is not accessible. This was based on Karwai's designs at the time. Will be revising.

File: assets/scss/styles.scss (hunk 5 above)
Comment (sassquad) [inline assets/scss/styles.scss]: Tom and I agree that using px based measurements is not accessible. This was based on Karwai's designs at the time. Will be revising.

File: assets/scss/styles.scss (hunk 6 above)
Comment (sassquad) [inline assets/scss/styles.scss:87]: Tom and I agree that using px based measurements is not accessible. This was based on Karwai's designs at the time. Will be revising.

File: assets/scss/styles.scss (hunk 7 above)
Comment (sassquad) [inline assets/scss/styles.scss]: Tom and I agree that using px based measurements is not accessible. This was based on Karwai's designs at the time. Will be revising.

File: assets/scss/styles.scss (hunk 8 above)
Comment (sassquad) [inline assets/scss/styles.scss]: Tom and I agree that using px based measurements is not accessible. This was based on Karwai's designs at the time. Will be revising.

File: assets/templates/stat.html (hunk 13 above)
Comment (sassquad) [inline assets/templates/stat.html]: Originally, the centre name was displayed inside this column. This has been moved outside of the table. The way the original design looked, I couldn't see a way of keeping the layout as a table, without introducing extra columns purely for presentational reasons, which isn't warranted.

For now, this column can be merged with the next one. Will be keen to discuss this layout with Tom.

**Comment by chrisns** [comment]
won't merge, needs css to be fixed, merge commits removed have split out the functional changes into another branch and PR, so the css+html work can be rebased from that work.

**Comment by chrisns** [comment]
new pull request at: https://github.com/UKHomeOffice/removals_dashboard/pull/13

**Pull request closed without merging**